source/PlayerInfo.cpp

```diff
@@ -2112,6 +2112,28 @@ void PlayerInfo::SelectNext()
 
 
 
+void PlayerInfo::DeselectAll()
+{
+	selectedWeapons.clear();
+}
+
+
+
+void PlayerInfo::ToggleAny(const Outfit *outfit)
```

Comment:
I'm not sure about the `Any` in this method name. Maybe `ToggleSecondary`?

Reply:
The method above it is called `SelectNext`, this seems inline with that to me.

Reply:
The other names are also not very descriptive. Let's set a good example in this PR and use a more descriptive name.

Reply:
SelectedWeapons -> SelectedSecondaryWeapons
SelectNext -> SelectNextSecondary
DeselectAll -> DeselectAllSecondaries
ToggleAny -> ToggleAnySecondary